package.json

```diff
@@ -13,7 +13,7 @@
     "start:frontend:https": "node aio/scripts/version.js && ng serve --progress=false --aot --proxy-config aio/https-proxy.conf.json --ssl --host=0.0.0.0",
```

Comment:
Could you also change `--host` parameter here and above to use this new variable?